cmd/query/app/http_handler.go

```diff
@@ -450,6 +451,11 @@ func (aH *APIHandler) writeJSON(w http.ResponseWriter, r *http.Request, response
 		}
 	}
 	resp, _ := marshall(response)
-	w.Header().Set("Content-Type", "application/json")
+
+	header := w.Header()
+	header.Set("Content-Type", "application/json")
+	for key, values := range aH.additionalHeaders {
+		header[key] = values
```

Comment:
should this be calling `header.Add()` instead?

Reply:
I debated both.   I ended up on this because the code is cleaner and, if any other headers besides `Content-Type` were hardcoded, the user would have the ability to overwrite them.

Either way is fine with me.  If you prefer `.Add()` I can swap it.

Reply:
Add will support repeated headers, which are valid in HTTP.

Reply:
The above supports repeated headers.  `http.Header` is a `map[string][]string`.  `values` is a `[]string` not just a `string`.